### PR TITLE
Merge sort

### PR DESCRIPTION
**Pull request by paronicholas**
- [x] Top-level README “Table of Contents” is updated
- [x] Feature tasks for this challenge are completed
- [x] Unit tests written and passing
    - [x] “Happy Path” - Expected outcome
    - [x] Expected failure
    - [x] Edge Case (if applicable/obvious)
- [x] README for this challenge is complete
    - [x] Summary, Description, Approach & Efficiency, Solution, Resources
    - [x] Link to code
    - [x] Link to tests

### CODE REVIEW

File: code401challenges/src/main/java/mergeSort/BLOG.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+#### Description
+The merge sort is a divide and conquer method. It utilizes a recursive method to divide an array into sub arrays until it is down to two elements and then checks to see which element is smaller than the other and sorts those against each other.
+
+The merge sort will use two methods:
+* `mergeSort(int[] arr, int len)`
+* `merge(int[] arr, int[] left, int[] right, int l, int r)`
+
+#### Effectiveness
+Time and Space:
+* Time: O(n)
```
Comment (evanslaton) [inline code401challenges/src/main/java/mergeSort/BLOG.md]: O(n log n)
